### PR TITLE
Api integration/therapy op 492

### PR DESCRIPTION
**Pull request by tsognong**

### CODE REVIEW

File: src/components/accessories/dateField/DateField.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ const DateField: FunctionComponent<IProps> = ({
         inputVariant="outlined"
         margin="dense"
         value={value}
+        //disablePast
```
Comment (almeida1492) [inline src/components/accessories/dateField/DateField.tsx:52]: Can we remove this comment?

File: src/components/accessories/patientBooking/bookingForm/BookingForm.tsx [hunk 2]
```diff
@@ -1,23 +1,100 @@
...
+  const initialValues = getFromFields(fields, "value");
+  const options = getFromFields(fields, "options");
+
+  const formik = useFormik({
+    initialValues,
+    validationSchema,
+    enableReinitialize: true,
+    onSubmit: (values) => {
+      const formattedValues = formatAllFieldValues(fields, values);
+      console.log("data", formattedValues);
```
Comment (almeida1492) [inline src/components/accessories/patientBooking/bookingForm/BookingForm.tsx:48]: Maybe we should remove this log.

File: src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx [hunk 3]
```diff
@@ -11,21 +14,34 @@ const PatientTherapyTable: FunctionComponent<IOwnProps> = ({
...
   const onDelete = () => {
     console.log("delete");
   };
 
+  const onEdit = () => {
+    console.log("update");
+  };
+
+  const onEView = () => {};
+
```
Comment (almeida1492) [inline src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx:44]: It looks like there's something missing here.

**Review by almeida1492 — CHANGES_REQUESTED** [review]

File: src/components/accessories/patientBooking/bookingForm/BookingForm.tsx (hunk 2 above)
Comment (tsognong) [reply src/components/accessories/patientBooking/bookingForm/BookingForm.tsx]: OK.

File: src/components/accessories/dateField/DateField.tsx (hunk 1 above)
Comment (tsognong) [reply src/components/accessories/dateField/DateField.tsx]: OK

File: src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx (hunk 3 above)
Comment (tsognong) [reply src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx:44]: I think for the table manipulation the common logic should be defined first. Or may be I am missing it.

File: src/components/accessories/dateField/DateField.tsx (hunk 1 above)
Comment (tsognong) [reply src/components/accessories/dateField/DateField.tsx]: OK

File: src/components/accessories/patientBooking/bookingForm/BookingForm.tsx (hunk 2 above)
Comment (tsognong) [reply src/components/accessories/patientBooking/bookingForm/BookingForm.tsx]: OK

File: src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx (hunk 3 above)
Comment (tsognong) [reply src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx:44]: At this moment we do not yet have a table manipulation common logic, or may be I am missing it

**Review by almeida1492 — APPROVED** [review]

**Pull request merged**